.github/workflows/bulk-dep-upgrades.yaml

```diff
@@ -0,0 +1,17 @@
+name: Upgrade dependencies
+on:
+  workflow_dispatch:
+  schedule:
+    # Runs 12:00AM on the first of every month
+    - cron: '0 0 1 * *'
+jobs:
+  upgrade:
+    # using `main` as the ref will keep your workflow up-to-date
+    uses: hashicorp/vault-workflows-common/.github/workflows/bulk-dependency-updates.yaml@main
+    secrets:
+      VAULT_ECO_GITHUB_TOKEN: ${{ secrets.VAULT_ECO_GITHUB_TOKEN }}
```

Comment:
@fairclothjm what's the process to get this secret added to this repo?

Reply:
Update: filed a helpdesk ticket

Reply:
That's it! Thanks!